### PR TITLE
Bug 1161095 - Don't prompt on theme-app uninstall.

### PR DESCRIPTION
**Pull request by etiennesegonzac**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1efa0cfd429201bd5486bbd368c503df2b632cf3

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3cfb78e94e98ba69b3350ccbfd24f721bc319795

File: apps/system/js/app_install_manager.js [hunk 1]
```diff
@@ -240,6 +240,11 @@ var AppInstallManager = {
     // Wrap manifest to get localized properties
     manifest = new ManifestHelper(manifest);
 
+    if (manifest.role == 'theme') {
```
Comment (KevinGrandon) [inline apps/system/js/app_install_manager.js]: Maybe ```===```?

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a5ffd8bfbc65fa58003853af3e3ecde4b52e05b8

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=10040b5595afcff2f26c4258862629955f12aed4

**Pull request merged**